src/styles/colors.scss

```diff
@@ -1,5 +1,9 @@
 $start-color: #ff313b;
-$finish-color: #1877ff;
+$finish-color: #007bff;
 $wall-color: #333333;
-$path-color: #16ff62;
-$option-color: #fff24e;
\ No newline at end of file
+$path-color: #af71db;
+$path-hover-color: #7e61c4;
+$selected-path-color: #1ace65;
+$choosable-option-color: #fff24e;
+$additional-color: #FF8833;
+$box-shadow-color: #727272;
```

Comment:
Maybe define the colors first in variable and then associate the color variables to functional variables. It's easier to change later on.